skrub/_dataframe/tests/test_common.py

```diff
@@ -165,6 +165,9 @@ def test_all_null_like(df_module):
         expected = expected.astype("bool")
     df_module.assert_column_equal(ns.is_null(col), expected)
 
+    col = ns.all_null_like(df_module.example_column, length=2)
```

Comment:
could we also have a test where the required length is bigger than the input column?

Reply:
Certainly ! 

I've added the option to create null columns longer than the input column, by removing `index=col.index` for `index=np.arange(length)`. We don't keep the index -- bu we weren't using it anywhere -- and now it's possible to create longer columns. I think it makes sense, but let me know if you'd rather have a max length set to `len(col)`